keyboards/handwired/yamsek/info.json

```diff
@@ -0,0 +1,77 @@
+                {"matrix":[4,4], "label":"R", "x":4, "y":1.125},
+                {"matrix":[4,5], "label":"T", "x":5, "y":1.25},
+                {"matrix":[5,0], "label":"Tab", "x":0, "y":2.75},
+                {"matrix":[5,1], "label":"A", "x":1, "y":2.375},
+                {"matrix":[5,2], "label":"S", "x":2, "y":2.125},
+                {"matrix":[5,3], "label":"D", "x":3, "y":2},
+                {"matrix":[5,4], "label":"F", "x":4, "y":2.125},
+                {"matrix":[5,5], "label":"G", "x":5, "y":2.25},
+                {"matrix":[6,0], "label":"LShift", "x":0, "y":3.75},
+                {"matrix":[6,1], "label":"Z", "x":1, "y":3.375},
+                {"matrix":[6,2], "label":"X", "x":2, "y":3.125},
+                {"matrix":[6,3], "label":"C", "x":3, "y":3},
+                {"matrix":[6,4], "label":"V", "x":4, "y":3.125},
+                {"matrix":[6,5], "label":"B", "x":5, "y":3.25},
+                {"matrix":[7,0], "label":"TG(Game)", "x":1, "y":6.25},
+                {"matrix":[7,1], "label":"Mo(NPad)", "x":0, "y":6.25},
+                {"matrix":[7,2], "label":"Backs", "x":0, "y":5.25},
+                {"matrix":[7,3], "label":"Alt", "x":1.2, "y":4.5},
+                {"matrix":[7,4], "label":"Ctrl", "x":2.25, "y":4.25},
+                {"matrix":[7,5], "label":"Shift", "x":3.25, "y":3.8},
+                {"matrix":[0,0], "label":"Y", "x":13, "y":1.25},
+                {"matrix":[0,1], "label":"U", "x":14, "y":1.125},
+                {"matrix":[0,2], "label":"I", "x":15, "y":1},
+                {"matrix":[0,3], "label":"O", "x":16, "y":1.125},
+                {"matrix":[0,4], "label":"P", "x":17, "y":1.375},
+                {"matrix":[0,5], "label":"-", "x":18, "y":1.75},
+                {"matrix":[1,0], "label":"H", "x":13, "y":2.25},
+                {"matrix":[1,1], "label":"J", "x":14, "y":2.125},
+                {"matrix":[1,2], "label":"K", "x":15, "y":2},
+                {"matrix":[1,3], "label":"L", "x":16, "y":2.125},
+                {"matrix":[1,4], "label":":", "x":17, "y":2.375},
+                {"matrix":[1,5], "label":"'", "x":18, "y":2.75},
+                {"matrix":[2,0], "label":"N", "x":13, "y":3.25},
+                {"matrix":[2,1], "label":"M", "x":14, "y":3.125},
+                {"matrix":[2,2], "label":"<", "x":15, "y":3},
+                {"matrix":[2,3], "label":">", "x":16, "y":3.125},
+                {"matrix":[2,4], "label":"?", "x":17, "y":3.375},
+                {"matrix":[2,5], "label":"=", "x":18, "y":3.75},
+                {"matrix":[3,0], "label":"Shift", "x":3.5, "y":3.85},
+                {"matrix":[3,1], "label":"Enter", "x":2.5, "y":4.3},
+                {"matrix":[3,2], "label":"Spac", "x":1.45, "y":4.55},
+                {"matrix":[3,3], "label":"Del", "x":0.25, "y":5.3},
+                {"matrix":[3,4], "label":"Mo(Arr)", "x":0.25, "y":6.3},
+                {"matrix":[3,5], "label":"Mo(Fun)", "x":1.25, "y":6.3},
```

Comment:
This still looks wrong — the order of entries here need to match this:
```c
    [BASE] = LAYOUT( // Base
        //left                                                                                   //right
        KC_ESC     ,KC_Q       ,KC_W       ,KC_E       ,KC_R       ,KC_T,                        KC_Y       ,KC_U       ,KC_I       ,KC_O       ,KC_P       ,KC_MINUS,
        KC_TAB     ,KC_A       ,KC_S       ,KC_D       ,KC_F       ,KC_G,                        KC_H       ,KC_J       ,KC_K       ,KC_L       ,KC_SCLN    ,KC_QUOT,
        KC_LSFT    ,KC_Z       ,KC_X       ,KC_C       ,KC_V       ,KC_B,                        KC_N       ,KC_M       ,KC_COMM    ,KC_DOT     ,KC_SLSH    ,KC_EQUAL,
        TG(GAME)   ,MO(NPAD)   ,KC_BSPC    ,KC_LALT    ,KC_LCTL    ,KC_LSFT,                     KC_RSFT    ,KC_ENT     ,KC_SPC     ,KC_ALDL    ,MO(ARRO)   ,MO(FUNC)
        ),
```
So instead of “the whole left half, then the whole right half” you need to have the entries ordered by rows across the keyboard as a whole — e.g., `Y`, `I`, `O`, `P` should go after `T`.

And coordinates of keys on the bottom row look wrong (the right side should have much larger `x` values).  You may look at the `qmk info -m` and `qmk info -l` output for your keyboard — it should resemble the physical appearance of the whole board (within some limitations).

Reply:
now i get it - thanks for the hint with 'qmk info' -> fixed with c0c59421f4

Reply:
Still not fixed completely — see below (but now GitHub consider these changes to be outdated, so I opened a new thread instead).